.github/workflows/i18n_check.yaml

```
@@ -0,0 +1,40 @@
+name: i18n Check
+on:
+  workflow_call: { }
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    permissions:
+      pull-requests: read
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: "Get modified files"
+        id: changed_files
+        if: github.event_name == 'pull_request' && github.event.pull_request.user.login != 'RiotTranslateBot'
+        uses: tj-actions/changed-files@v19
+        with:
+          files: |
+            src/i18n/strings/*
+          files_ignore: |
+            src/i18n/strings/en_EN.json
+
+      - name: "Assert only en_EN was modified"
+        if: |
+          github.event_name == 'pull_request' &&
+          github.event.pull_request.user.login != 'RiotTranslateBot' &&
+          steps.changed_files.outputs.any_modified == 'true'
+        run: |
+          echo "You can only modify en_EN.json, do not touch any of the other i18n files as Weblate will be confused"
```

I'm aware this is a copy/paste, but might be worth adding developer hints:

```suggestion
          echo "Only translation files modified by `yarn i18n` can be committed - other translation files will confuse weblate in unrecoverable ways."
```